app/serializers/serialize_submission.rb

```diff
@@ -6,7 +6,15 @@ class SerializeSubmission
   def call
     {
       uuid: submission.uuid,
-      tests_status: submission.tests_status
+      tests_status: submission.tests_status,
+      links: {
```

Comment:
@iHiD Want to point you towards this. I've added the link to cancel in the submission serializer, rather than constructing the URL in the client side to make the front end and back end less coupled. Let me know what you think.

Reply:
I like that, as it will be helpful when we'll be building external integrations (like IDE plugins).

Reply:
Yep - I'm happy with this too.